### PR TITLE
DO-630 and fix bug in link to DataPlatform

### PR DESCRIPTION
**Pull request by AlenaYanish**

### CODE REVIEW

File: landing/static/js/index.js [hunk 1]
```diff
@@ -114,25 +155,34 @@ $('#contact-form').submit(function(event){
     }
 
     $.ajax({
-        url: "https://ipa.dataocean.us/api/landing_mail/",
+        url: "https://dataocean-ipa.ml/api/landing_mail/",
```
Comment (roman-tiukh) [inline landing/static/js/index.js:158]: host should be a environment variable, for test its have one value for prod other.

File: landing/static/js/index.js [hunk 2]
```diff
@@ -114,25 +155,34 @@ $('#contact-form').submit(function(event){
...
         data: data,
         success: function(data, status, xhr) {
             if (xhr.status !== 200) {
                 return
             }
-            alert('Тепер Ви будете в курсі всіх новин про DataOcean!');
+            if (userLang === 'uk') {
+                alert('Тепер Ви будете в курсі всіх новин про DataOcean!');
+            } else 
+                alert('Now you will be able to keep up with all of DataOcean updates!');
```
Comment (roman-tiukh) [inline landing/static/js/index.js:169]: create a collection with messages, as I told you)

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: landing/static/js/index.js [hunk 3]
```diff
@@ -141,8 +202,18 @@ function changeLanguage (langCode) {
...
                 $(this).show();
+                if (langCode === 'uk') {
+                    $('#name')[0].placeholder = 'Петро';
+                    $('#surname')[0].placeholder = 'Іваненко';
+                    $('#question')[0].placeholder = 'Привіт, Data Ocean! Я хотів запитати...';
+                } else {
+                    $('#name')[0].placeholder = 'John';
+                    $('#surname')[0].placeholder = 'Galt';
+                    $('#question')[0].placeholder = 'Hello, Data Ocean! I would like to ask about...';
+                }
```
Comment (roman-tiukh) [inline landing/static/js/index.js:215]: translations here

File: landing/static/js/index.js (hunk 1 above)
Comment (roman-tiukh) [reply landing/static/js/index.js:178]: this is actual problem, you can change to https://ipa.dataocean.us temporary

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: landing/static/js/index.js [hunk 4]
```diff
@@ -114,25 +187,25 @@ $('#contact-form').submit(function(event){
     }
 
     $.ajax({
-        url: "https://ipa.dataocean.us/api/landing_mail/",
+        url: "https://ipa.dataocean.us",
```
Comment (roman-tiukh) [inline landing/static/js/index.js:190]: @AlenaYanish ......

File: landing/static/js/index.js [hunk 5]
```diff
@@ -141,10 +214,15 @@ function changeLanguage (langCode) {
     $('#select_language').val(langCode);
     if (allowedLanguages.includes(langCode)) {
         $("[lang]").each(function () {
-            if ($(this).attr("lang") === langCode)
+            if ($(this).attr("lang") === langCode) {
                 $(this).show();
-            else
+                $('#name')[0].placeholder = t('placeholderName');
+                $('#surname')[0].placeholder = t('placeholderLastName');
+                $('#question')[0].placeholder = t('placeholderQuestion');
```
Comment (roman-tiukh) [inline landing/static/js/index.js:221]: this should located in other place

File: landing/static/js/index.js [hunk 6]
```diff
@@ -52,59 +52,132 @@ $(document).ready(() => {
...
-}
+    }
+};
 
 $('#contact-form').submit(function(event){
     event.preventDefault();
-    if (!$(this).validate(contactFormScheme)) {
-        return
+    if (!$(this).validate(getSchema())) {
+         return $(this).parents
```
Comment (roman-tiukh) [inline landing/static/js/index.js:178]: what is $(this).parents

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: landing/static/js/index.js [hunk 7]
```diff
@@ -187,7 +187,7 @@ $('#contact-form').submit(function(event){
     }
 
     $.ajax({
-        url: "https://ipa.dataocean.us",
+        url: "process.env.PLATFORM_HOST",
```
Comment (roman-tiukh) [inline landing/static/js/index.js]: ... wrong link

File: landing/.env.example [hunk 8]
```diff
@@ -0,0 +1,4 @@
+### LOCAL VARS
+# local variables, you can override them in ".env" file
+
+DO_BACKEND_HOST, DO_FRONTEND_HOST
```
Comment (roman-tiukh) [inline landing/.env.example:4]:
```suggestion
DO_BACKEND_HOST=https://ipa.dataocean.us
DO_FRONTEND_HOST=https://dp.dataocean.us
```

File: landing/.gitignore [hunk 9]
```diff
@@ -0,0 +1 @@
+/.env
```
Comment (roman-tiukh) [inline landing/.gitignore:1]:
```suggestion
/.env

```

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: landing/static/js/index.js (hunk 7 above)
Comment (AlenaYanish) [reply landing/static/js/index.js]: I fix it in "add env vars" commit

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**